devlab/tests/generate_load.py

```diff
@@ -1032,6 +1035,17 @@ def break_image(self):
             self.migration_utils.execute_command_on_vm(
                 self.get_vagrant_vm_ip(), cmd, username='root', password='')
 
+    def create_network_with_segm_id(self):
+        self.dst_cloud = Prerequisites(
```

Comment:
We already have Prerequisites instance for dst_cloud (https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/devlab/tests/generate_load.py#L950)